### PR TITLE
Unit tests modules/accounts

### PR DESCRIPTION
**Pull request by roman-bystrimovich**

### CODE REVIEW

File: tests/unit/modules/accounts.spec.js [hunk 1]
```diff
@@ -0,0 +1,1889 @@
...
+
+        expect(scope.logic.transaction.attachAssetType.calledOnce).to.be.true;
+
+        expect(scope.logic.transaction.attachAssetType.getCall(0).args.length).to.equal(2);
+        expect(scope.logic.transaction.attachAssetType.getCall(0).args[0]).to.equal(transactionTypes.VOTE);
+        expect(scope.logic.transaction.attachAssetType.getCall(0).args[1]).to.be.an.instanceof(Vote);
+
+        expect(callback.called).to.be.false;
+        setImmediate(function() {
+            expect(callback.called).to.be.true;
```
Comment (nerdvibe) [inline tests/unit/modules/accounts.spec.js]: Isn't this redundant?

File: tests/unit/modules/accounts.spec.js [hunk 2]
```diff
@@ -0,0 +1,1889 @@
...
+            account.getAccount({ address : testAccount.address }, testObj1, callback);
+
+            expect(scope.logic.account.get.calledOnce).to.be.true;
+            expect(scope.logic.account.get.getCall(0).args.length).to.equal(3);
+            expect(scope.logic.account.get.getCall(0).args[0]).to.deep.equal({ address : testAccount.address });
+            expect(scope.logic.account.get.getCall(0).args[1]).to.equal(testObj1);
+            expect(scope.logic.account.get.getCall(0).args[2]).to.equal(callback);
+        });
+
+        it("public.getAccounts", function() {
```
Comment (nerdvibe) [inline tests/unit/modules/accounts.spec.js]: I was a bit confused, shouldn't this be in a separate describe?

File: tests/unit/modules/accounts.spec.js [hunk 3]
```diff
@@ -0,0 +1,1889 @@
...
+            request.body.publicKey = secondAccount.publickKey;
+            request.body.multisigAccountPublicKey = firstAccount.publicKey;
+
+            modules.accounts.getAccount.onCall(0).callsArgWith(1, null, firstAccount);
+            modules.accounts.getAccount.onCall(1).callsArgWith(1, null, secondAccount);
+            scope.logic.transaction.create.throws(error)
+
+            account.shared.addDelegates(request, callback);
+
+            expect(scope.schema.validate.calledOnce).to.be.
```
Comment (nerdvibe) [inline tests/unit/modules/accounts.spec.js]: broken indentation...

**Review by nerdvibe — CHANGES_REQUESTED** [review]
I believe I've already reviewed this PR, but here is my review :)

Nice PR, I've left some comments.

Also, if I'm not wrong the `setAccountAndGet` and `isLoaded` are not covered in the test. Is this on purpose?

**Comment by roman-bystrimovich** [comment]
@xunga fixed

**Review by nerdvibe — APPROVED** [review]

**Comment by nerdvibe** [comment]
@roman-bystrimovich all righty! Then for me it's a 👍

**Comment by vekexasia** [comment]
@roman-bystrimovich can you please commit something into this so that travis can check against developmnet and code coverage?

**Pull request merged**